src/features/CustomViews.ts

```diff
@@ -2,8 +2,9 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
+import path = require("path");
 import vscode = require("vscode");
```

Comment:
```suggestion
import vscode from 'vscode';
```

Reply:
`import * as vscode from 'vscode'` (for clarity)